### PR TITLE
v5.12.3

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,56 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.12.3
+
+_Jun 23, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Swedish (sv-SE) locale on the data grid (#5210) @mrxdst
```
Comment (flaviendelangle) [inline CHANGELOG.md:12]: If we keep the highlight together we should start to say which product has the new locale

Comment (alexfauquette) [reply CHANGELOG.md:12]: Here it's not a problem because he did both the pickers and the data grid at once 🙀😄

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,56 @@
...
+- [pickers] Export adapters from both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` (#5204) @flaviendelangle
+- [pickers] Add Dutch (nl-NL) locale (#5237) @DDukers
+
+### Docs
+
+- [docs] Add `DataGridPremium` to all API sections (#5196) @flaviendelangle
+- [docs] Add plan badge next to event name in the Events page (#5200) @flaviendelangle
+- [docs] Add section for theme augmentation in the picker doc (#5276) @flaviendelangle
+- [docs] Add waiting for upvote section for row group panel (#5271) @flaviendelangle
+- [docs] Fix typo in the `DateRangePicker` documentation' (#5259) @flaviendelangle
```
Comment (cherniavskii) [inline CHANGELOG.md:42]:
```suggestion
- [docs] Fix typo in the `DateRangePicker` documentation (#5259) @flaviendelangle
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,56 @@
...
+
+### Core
+
+- [core] Allows having multiple playgrounds (#5288) @alexfauquette
```
Comment (cherniavskii) [inline CHANGELOG.md:46]:
```suggestion
- [core] Allow having multiple playgrounds (#5288) @alexfauquette
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,56 @@
...
+- [core] Improve typing of `GridFilterInputMultipleSingleSelect` (#5206) @flaviendelangle
+- [core] Remove arbitraries new lines (#5245) @oliviertassinari
```
Comment (cherniavskii) [inline CHANGELOG.md:48]:
```suggestion
- [core] Remove arbitrary new lines (#5245) @oliviertassinari
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,56 @@
...
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.12.3` / `@mui/x-data-grid-pro@v5.12.3` / `@mui/x-data-grid-premium@v5.12.3`
+
+#### Changes
+
+- [DataGrid] Do not regenerate the row tree when `props.loading` changes (#5213) @flaviendelangle
+- [DataGrid] Fix the default filter operator fallback on state initialization (#5266) @flaviendelangle
+- [DataGrid] Stop using the deprecated api prop in the grid components (#5205) @flaviendelangle
```
Comment (cherniavskii) [inline CHANGELOG.md:23]:
```suggestion
- [DataGrid] Stop using the deprecated `api` prop in the grid components (#5205) @flaviendelangle
```

**Review by cherniavskii — APPROVED** [review]

**Review by m4theushw — APPROVED** [review]
Two suggestions but it's already good to release.

**Review by alexfauquette — APPROVED** [review]

**Review by DanailH — APPROVED** [review]

**Pull request merged**